tests/api/test_chip.cpp

```diff
@@ -44,12 +44,16 @@ inline std::unique_ptr<tt_ClusterDescriptor> get_cluster_desc() {
         return nullptr;
     }
 
-    // TODO: remove getting manually cluster descriptor from yaml.
-    std::string yaml_path = test_utils::GetClusterDescYAML();
+    // TODO: Remove different branch for different archs
     std::unique_ptr<tt_ClusterDescriptor> cluster_desc;
     if (device_arch == tt::ARCH::GRAYSKULL) {
         cluster_desc = tt_ClusterDescriptor::create_for_grayskull_cluster(pci_device_ids_set, pci_device_ids);
+    } else if (device_arch == tt::ARCH::BLACKHOLE) {
+        std::string yaml_path = test_utils::GetAbsPath("blackhole_1chip_cluster.yaml");
+        cluster_desc = tt_ClusterDescriptor::create_from_yaml(yaml_path);
     } else {
```

Comment:
would explicit 

```cpp
if (device_arch == tt::ARCH::WORMHOLE_B0)
```

be more readable here? Same for all other changes

Reply:
Well actually not. The else branch which is following is not "something specific for wormhole", rather it is a "standard" way of doing things. Grayskull and Blackhole have something specific done to them. 
To be specific, in one of the following PRs (specifically when cem starts working for BH) I'd expect to remove the blackhole branch. Maybe in some of the following changes to remove Grayskull branch as well.